src/config/global-config.c

```diff
@@ -55,6 +56,8 @@ int Read_GlobalSK(XML_NODE node, void *configp, __attribute__((unused)) void *ma
             merror(XML_VALUENULL, node[i]->element);
             return (OS_INVALID);
         } else if (strcmp(node[i]->element, xml_auto_ignore) == 0) {
+            mwarn("The <syscheck><%s> tag is deprecated, please use <global><fim><%s> instead.", xml_auto_ignore, xml_auto_ignore);
```

Comment:
Warnings about deprecated options are being logged for three times each start-up:

```
2019/02/05 07:01:46 ossec-testrule: WARNING: The <syscheck><auto_ignore> tag is deprecated, please use <global><fim><auto_ignore> instead.
2019/02/05 07:01:46 ossec-maild: WARNING: The <syscheck><auto_ignore> tag is deprecated, please use <global><fim><auto_ignore> instead.
2019/02/05 07:01:46 ossec-analysisd: WARNING: The <syscheck><auto_ignore> tag is deprecated, please use <global><fim><auto_ignore> instead.